workflows/python/sdk/README.md

```diff
@@ -0,0 +1,72 @@
+# Dapr workflows
+
+In this quickstart, you'll create a simple console application to demonstrate Dapr's workflow programming model and the workflow management API. The console app starts and manages the lifecycle of a workflow that stores and retrieves data in a state store.
+
+This quickstart includes one project:
+
+- python console app `order-processor` 
+
+The quickstart contains 1 workflow order_processing_workflow to simulate purchasing items from a store, and 5 unique activities within the workflow. These 5 activities are as follows:
+
+- notify_activity: This activity utilizes a logger to print out messages throughout the workflow. These messages notify the user when there is insufficient inventory, their payment couldn't be processed, and more.
+- process_payment_activity: This activity is responsible for processing and authorizing the payment.
+- verify_inventory_activity: This activity checks the state store to ensure that there is enough inventory present for purchase.
+- update_inventory_activity: This activity removes the requested items from the state store and updates the store with the new remaining inventory value.
+- requst_approval_activity: This activity seeks approval from Manager, if payment is greater than 50000 USD.
+
+### Run the order processor workflow
+
+1. Open a new terminal window and navigate to `order-processor` directory: 
+
+<!-- STEP
+name: Install requirements
+-->
+
+```sh
+cd ./order-processor
+pip3 install -r requirements.txt
+```
+
+<!-- END_STEP -->
+2. Run the console app with Dapr: 
+<!-- STEP
+name: Running this example
+expected_stdout_lines:
+  - "There are now 89 cars left in stock"
+  - "Purchase of item is  COMPLETED"
+output_match_mode: substring
+background: true
+timeout_seconds: 30
+sleep: 15
+-->
+
+```sh
+cd ./order-processor
+dapr run --app-id order-processor --app-protocol grpc --dapr-grpc-port 4001 --components-path ../../../components/ --placement-host-address localhost:50005 -- python3 app.py
```

Comment:
Should we instead be using the default gRPC port?

Reply:
@DeepanshuA  is there a reason why we're not using the standard gRPC port?

Reply:
Had used 4001 before https://github.com/dapr/python-sdk/pull/570 was merged. Now, can use any port.

Reply:
`placement-host-address` is required while debugging on local. I have removed it from here.